gax/src/grpc.ts

```diff
@@ -398,14 +400,29 @@ export class GrpcClient {
       'grpc.channelFactoryOverride',
       'grpc.gcpApiConfig',
     ];
-    const [cert, key] = await this._detectClientCertificate(options);
+    const [cert, key] = await this._detectClientCertificate(
+      options,
+      options.universeDomain
+    );
     const servicePath = this._mtlsServicePath(
       options.servicePath,
       customServicePath,
       cert && key
     );
     const opts = Object.assign({}, options, {cert, key, servicePath});
     const serviceAddress = servicePath + ':' + opts.port;
+    if (!options.universeDomain) {
+      options.universeDomain = 'googleapis.com';
+    }
+    if (options.universeDomain) {
+      const universeFromAuth = await this.auth.getUniverseDomain();
+      if (universeFromAuth && options.universeDomain !== universeFromAuth) {
+        throw new Error(
+          `The configured universe domain (${options.universeDomain}) does not match the universe domain found in the credentials (${universeFromAuth}). ` +
+            "If you haven't configured the universe domain explicitly, googleapis.com is the default."
+        );
+      }
+    }
```

Comment:
`await this.auth.getUniverseDomain()` is called even if there is no universe domain configured in the options (because it then falls back to `googleapis.com`) which takes a considerable amount of time. In my case around 300ms. Should it really work that way? @alexander-fenster 

This change slows down my project testing setup a lot because each test-bigtable creation takes around 300ms longer.